test/e2e/apimachinery/webhook.go

```diff
@@ -244,6 +247,251 @@ var _ = SIGDescribe("AdmissionWebhook", func() {
 		slowWebhookCleanup()
 	})
 
+	ginkgo.It("patching/updating a validating webhook should work", func() {
+		client := f.ClientSet
+		admissionClient := client.AdmissionregistrationV1()
+
+		ginkgo.By("Creating a validating webhook configuration")
+		hook, err := createValidatingWebhookConfiguration(f, &admissionregistrationv1.ValidatingWebhookConfiguration{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: f.UniqueName,
+			},
+			Webhooks: []admissionregistrationv1.ValidatingWebhook{
+				newDenyConfigMapWebhookFixture(f, context),
+			},
+		})
+		framework.ExpectNoError(err, "Creating validating webhook configuration")
+		defer func() {
+			err := client.AdmissionregistrationV1().ValidatingWebhookConfigurations().Delete(hook.Name, nil)
+			framework.ExpectNoError(err, "Deleting validating webhook configuration")
+		}()
+		ginkgo.By("Creating a configMap that does not comply to the validation webhook rules")
+		err = wait.PollImmediate(100*time.Millisecond, 30*time.Second, func() (bool, error) {
+			cm := namedNonCompliantConfigMap(string(uuid.NewUUID()), f)
+			_, err = client.CoreV1().ConfigMaps(f.Namespace.Name).Create(cm)
+			if err == nil {
+				return false, nil
```

Comment:
nit, attempt a delete of the configmap here (ignoring errors)? just so we don't accumulate a ton of configmaps while waiting for the webhook